share/spice/code_search/code_search.js

```diff
@@ -9,18 +9,23 @@
 
```

Comment:
Can you update the error checking on [L4](https://github.com/duckduckgo/zeroclickinfo-spice/pull/1822/files#diff-57066020d432d64fc9b32ace072e0c3dR4). Right now it doesn't call `Spice.failed` when no results are returned.
Due to the statement **`!api_result.results.length === 0`**

This can be updated to  **`api_result.results.length === 0`** or check the **`api_result.total === 0`** either way :+1: